services/workflows-service/scripts/workflows/kyc-email-process-example.ts

```diff
@@ -13,6 +13,7 @@ export const kycEmailSessionDefinition = {
     initial: 'idle',
     states: {
       idle: {
+        tags: [StateTag.PENDING_PROCESS],
```

Comment:
`PENDING_PROCESS` signifies an in-progress process with a vendor. Is that the case for `idle`?

Reply:
in this context, the StateTag.PENDING_PROCESS of end user over KYC session ( this state is pending for getting kyc session )

Reply:
But is it pending a response from a 3rd party?